tests/components/alexa/__init__.py

```diff
@@ -171,6 +171,12 @@ def __init__(self, properties):
         """Initialize class."""
         self.properties = properties
 
+    def assert_not_has_property(self, namespace, name):
```

Comment:
This appears to only be used by the tests... should it be moved to a module in the tests (i.e. common or utilities)?

Reply:
It's in the tests folder already 😄

Reply:
Haha I must have been tired!